src/base/utilities.ts

```diff
@@ -1058,7 +1058,8 @@ module Shumway {
         d = chunk & 63; // 63 = 2^6 - 1
 
         // Convert the raw binary segments to the appropriate ASCII encoding
-        base64 += encodings[a] + encodings[b] + encodings[c] + encodings[d];
+        base64 +=
+            concat4(encodings[a], encodings[b], encodings[c], encodings[d]);
```

Comment:
Nit: we wrap arguments to the function call in cases like this. No need to change, will fix before merging.